package.json

```diff
@@ -8,6 +8,7 @@
   "repository": "meandavejustice/min-vid",
   "main": "index.js",
   "author": "meandave",
+  "updateURL": "https://testpilot.firefox.com/files/min-vid/updates.json",
```

Comment:
rdf is old and busted.  .json is new and hot.  Also, please get rid of the updateLink you added - that'll come from the json (and I'll add a checksum to it).